src/verify_expr.ml

```diff
@@ -1595,7 +1595,7 @@ module VerifyExpr(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
           Default -> cont h env (Some (match tp with PtrType _ -> null_pointer_term () | Bool -> false_term | _ -> ctxt#mk_intlit 0))
         | Expr e -> eval_h h env e $. fun h env value -> cont h env (Some value)
         | Unspecified -> cont h env None
-        | Term t -> cont h env (Some t)
+        | Term t | MaybeUninitTerm t -> cont h env (Some t)

```

Comment:
This doesn't make sense to me. Please explain? It seems better to simply not handle the `MaybeUninitTerm` case here?

Reply:
In Rust, `produce_points_to()` is used when closing a struct that is not a member of `structmap`. Hence, `MaybeUninitTern (u8s__of(...))` is passed to `produce_c_object` when closing it.

Reply:
But the code you're proposing will in that case produce a non-underscore (initialized) points-to chunk whose value is a ctor_opt value? That seems wrong? (It would also be wrong to call produce_points_to_chunk_ with a points_to_kind = MaybeUninit because then the value should be an `option<S>` (i.e. either `none` or fully initialized) , not a ctor_opt value.) The easy fix seems to be to continue calling produce_points_to_chunk_ with value `None` (that's OCaml's None) in this case for now.